op-plasma/cmd/daserver/flags.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+)
+
+const (
+	ListenAddrFlagName        = "addr"
+	PortFlagName              = "port"
+	S3BucketFlagName          = "s3.bucket"
+	S3EndpointFlagName        = "s3.endpoint"
+	S3AccessKeyIDFlagName     = "s3.access-key-id"
+	S3AccessKeySecretFlagName = "s3.access-key-secret"
+	FileStorePathFlagName     = "file.path"
+)
+
+const EnvVarPrefix = "OP_PLASMA_DA_SERVER"
+
+func prefixEnvVars(name string) []string {
+	return opservice.PrefixEnvVar(EnvVarPrefix, name)
+}
+
+var (
+	ListenAddrFlag = &cli.StringFlag{
+		Name:    ListenAddrFlagName,
+		Usage:   "rpc listening address",
+		Value:   "0.0.0.0",
```

Comment:
```suggestion
		Value:   "127.0.0.1",
```
If this is a private RPC, we'd like to make it localhost by default.